cypress/integration/channel/message_emoji_jumbo_spec.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 4]*/
+
+const normalSize = '21px';
+const jumboSize = '32px';
+
+const testCases = [
+    {message: 'This is a normal reply with emoji :smile: {enter}', emojiSize: normalSize},
+    {message: ':smile: {enter}', emojiSize: jumboSize},
+    {message: ':smile: :yum: {enter}', emojiSize: jumboSize},
+];
+
+function verifyLastPostStyle(expectedSize) {
+    //  * Verify size of the emoji
+    cy.getLastPostId().then(() => {
+        cy.get('span.emoticon').last().should('have.css', 'height', expectedSize).and('have.css', 'width', expectedSize);
```

Comment:
Oh, I totally missed it. Last 2 comments from me:
- Test requirement is for reply thread (or RHS).  However, this selector targets the one on center post (see screenshot).
- This (`last()`) only checks whether the last emoji is jumbo.  However, it should check whether all emojis in the last reply post are all jumbo.
Please take a look again, thanks

![Screen Shot 2019-06-26 at 3 01 36 PM](https://user-images.githubusercontent.com/5334504/60158462-c05b5880-9823-11e9-9314-ad48149f3c7e.png)